src/cards_management.c

```diff
@@ -87,10 +88,10 @@ int initialize_cards(void)
 }
 
 /**
- * @brief Deals each player 5 cards at the start of the game setup
+ * @brief Deals each player 5 cards during the game initialization
```

Comment:
Please update the corresponding description in the header file as well.

Reply:
Done